src/platform/qpg6100/CHIPDevicePlatformConfig.h

```diff
@@ -51,11 +51,11 @@
 // ========== Platform-specific Configuration Overrides =========
 
 #ifndef CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE
-#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE (3 * 1024 + 256)
+#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE (4 * 1024)
```

Comment:
Is there a relevance to 4 here?  Is this related to CHIP_CONFIG_MAX_DEVICE_ADMINS?  If so, should we perhaps have:

```suggestion
#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE (CHIP_CONFIG_MAX_DEVICE_ADMINS * 1024)
```

Else, why not just set the #define to 4096?  I'm not sure what value there is in showing the arithmetic if the reader doesn't know where it came from.

Reply:
The calculation is there for readability purposes - calculating in kB boundaries.
Currently we lack heap tracking support to fully fine-tune.

The ADMIN config is already to optimize the coming OpCreds addition in terms of flash and RAM buffers used per admin.